src/stacks-hierarchy/RootStack.tsx

```diff
@@ -168,6 +170,16 @@ export const RootStack = () => {
     };
   }
 
+  const isCurrentAppVersionLowerThanMinVersion =
+    compareVersion(APP_VERSION, minimum_app_version) > 0;
```

Comment:
```suggestion
    compareVersion(APP_VERSION, minimum_app_version) < 0;
```
I think you meant this way 😄

Reply:
Good point 🙈